src/pages/sign-up/SignupForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { signUp } from '@/apis/signup';
+import * as styles from './SignupStyles';
+
+type Props = {};
+
+function SignUpForm({}: Props) {
```

Comment:
P3 : 저번에 정한 컴포넌트 선언 컨벤션으로 함수 표현식을 사용으로 결정했는데 이번 컴포넌트는 선언문으로 선언 되어있네요.
```suggestion
const SignUpForm = () => {
```